django_lti_login/management/commands/add_lti_key.py

```diff
@@ -10,9 +10,9 @@ def add_arguments(self, parser):
         parser.add_argument('-f', '--force', action='store_true',
                             help="Overwrite existing keys")
         parser.add_argument('-k', '--key',
-                            help="The key. Only alphanumerals are allowed. Default: generate random")
+                            help="The key. Only alphanumericals are allowed. Default: generate random")
```

Comment:
I think the plural of "alphanumeric" is "alphanumerics". "Alphanumerical" is an adjective. :)
https://en.wiktionary.org/wiki/alphanumerics#English

Not a major issue, though.

Reply:
I was basing the change on [this](https://en.wikipedia.org/wiki/Alphanumeric):
> Alphanumericals are a combination of alphabetical and numerical characters, and is used to describe the collection of Latin letters and Arabic digits or a text constructed from this collection.

Though it seems that the article uses both "alphanumericals" and "alphanumerics", and it's only Wikipedia after all